usecases/traverser/explorer.go

```diff
@@ -397,6 +397,8 @@ func (e *Explorer) Concepts(ctx context.Context,
 
 func (e *Explorer) appendResultsIfSimilarityThresholdMet(item search.Result,
 	results *[]search.Result, vec []float32, params ExploreParams) error {
+	// TODO: The distancer should no longer be needed because we should get the
+	// raw distances as part of the search.Result
 	dist, err := e.distancer(vec, item.Vector)
```

Comment:
@parkerduckworth The other changes look really great now, great work.

Let's quickly discuss this one when you are in later today. If I interpret this correctly, this would mean that the hardcoded-always-cosine distancer is still in use for an `Explore {}` search (I only added e2e test for `Get {}`. 

This then raises a second question about whether Explore is ever feasible when more than one distance metric is used (see Slack).